engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -516,14 +541,126 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
         blockItem.send(new DropItemEvent(spawnPos));
         return "Spawned block.";
     }
+    @Command(shortDescription = "Drops many blocks as per value choosen by the player", helpText = "the particular block can be seleted which the player wants to drop"
+                , runOnServer = true, requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public String bulkDropper(@Sender EntityRef sender, @CommandParam("blockName") String blockName, @CommandParam("value") int value) {
+
+        //This is a loop which gives the particular amount of block the player wants to spawn
+        ClientComponent clientComponent = sender.getComponent(ClientComponent.class);
+        LocationComponent characterLocation = clientComponent.character.getComponent(LocationComponent.class);
+
+        Vector3f spawnPos = characterLocation.getWorldPosition();
+        Vector3f offset = characterLocation.getWorldDirection();
+        offset.scale(3);
+        spawnPos.add(5,10,0);
+        for (int i = 0; i <= value; i++) {
+            BlockFamily block = blockManager.getBlockFamily(blockName);
+            if (block == null) {
+                return "";
+            }
+            BlockItemFactory blockItemFactory = new BlockItemFactory(entityManager);
+            EntityRef blockItem = blockItemFactory.newInstance(block);
+
+            blockItem.send(new DropItemEvent(spawnPos));
+        }
+
+        // this returns the block you have spawned and the amount
+        return "Dropped " + value + " Blocks :)";
+    }
+
+    @Command(shortDescription = "Drops bowling pin", helpText = " `prepare` command to be entered along with the `block name` ",
```

Comment:
This description is incorrect, it doesn't drop a bowling pin. It arranges the blocks in a bowling pin format.
See the other commands for how the `helpText` section should be written as well.